zcutil/build-mac-dtest.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
@SirSevenG Why do we need these files - just to export compilation flag?

Reply:
Don't want to change build scripts, and only linux script `build.sh`adds to ./configure compilation flags from ENV.